save-points/05-checkout-with-validation/BlazingPizza.Server/OrdersController.cs

```diff
@@ -23,7 +23,7 @@ public OrdersController(PizzaStoreContext db)
     public async Task<ActionResult<List<OrderWithStatus>>> GetOrders()
     {
         var orders = await _db.Orders
-                .Where(o => o.UserId == GetUserId())
+                //.Where(o => o.UserId == GetUserId())
```

Comment:
Documentation states student has to uncomment these lines, but the existing implementation already have them commented out